snuba/migrations/snuba_migrations/transactions/0002_transactions_onpremise_compatibility.py

```diff
@@ -0,0 +1,179 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import (
+    Column,
+    DateTime,
+    LowCardinality,
+    Materialized,
+    Nullable,
+    String,
+    UInt,
+    WithDefault,
+)
+from snuba.clusters.cluster import ClickhouseClientSettings, get_cluster
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+
+
+UNKNOWN_SPAN_STATUS = 2
+
+
+def recreate_table() -> None:
```

Comment:
Actually there  are some additional changes we made to the table, starting in january, that would require a table recreation. The sampling key was the only critical one because it prevents the product from working otherwise. 
Are you going to apply the others as well ?

This was the PR:
https://github.com/getsentry/snuba/pull/692/files#diff-99e8c1b63173e867092f9d19e3bebdcbR197-R198

Reply:
The only thing that is not there is ensuring we delete _start_date and _finish_date, since these are not in the existing migration statement. I'll add them now. All the other changes are being applied in subsequent operations.

Reply:
Actually looks like partition by also changed. Will update.